src/@next/Popover/PopoverStyle.ts

```diff
@@ -145,7 +145,7 @@ export const StyledPopover: any = createGlobalStyle`
 
 .Polaris-PositionedOverlay {
   position: absolute;
-  z-index: 400;
+  z-index: 999;
```

Comment:
@farhanabi can we add a prop for a zIndexOverride instead? This will probably break some components that are using Popover.